tasks/rke2.yml

```diff
@@ -166,4 +272,6 @@
     owner: root
     group: root
     mode: 0644
-  when: rke2_environment_options is defined and rke2_environment_options|length > 0
+  when:
+    - rke2_environment_options is defined
+    - rke2_environment_options|length > 0
```

Comment:
Please remove the empty line

Reply:
thanks for the link! it fails because there is no empty line at the end of file - so I added one.